### DISCUSSION TITLE
When should the close tab icon be visible? (please vote)

### DISCUSSION THREAD

**Discussion by rakleed** [rustdesk/rustdesk#3363] [Polls]
Now for new users it is not at all obvious how to close the settings or connection tab, since the close icon only appears when hovering over the tab. This is implemented in many macOS applications, but it seems inconvenient to me. Much more I like the option when the close icon is always visible, especially since it does not take up much space (this is done in Chrome and many applications on Windows). [@rustdesk also suggests](https://github.com/rustdesk/rustdesk/discussions/3304#discussioncomment-5059253) another option where only the active tab always has a close icon, and only when hovering over a tab for inactive tabs. What do you think?

**Comment by deep-soft** [comment]
There should be four options:
all three from above
and
() Do not show close icon

In that case window can be closed with right click and Close option (first option on popup).

And the preferred option to be selectable in Settings, in General.
